cstar/tests/unit_tests/base/test_nothing.py

```diff
@@ -0,0 +1,2 @@
+def test_nothing():
+    pass
```

Comment:
What's the purpose of this?

Reply:
> It adds an empty unit test module as a placeholder, but this will be removed when we start adding others.

Github workflow fails on test collection without something to collect

Reply:
Riight sorry.